config/clients/js/CHANGELOG.md.mustache

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.3.0
+
+### [0.3.0](https://{{gitHost}}/{{gitUserId}}/{{gitRepoId}}/compare/v0.2.10...v0.3.0) (2023-11-01)
+
+- feat!: support for conditions
+
```

Comment:
any breaking change (i.e., use of WriteRequestTupleKey vs. ClientTupleKey)?
Do we also need to update our README example?